models/cloudship/cloudship.sdf

```diff
@@ -0,0 +1,649 @@
+
```

Comment:
If the sdf is generated with a jinja template, I think this file should not be commited